c++.html.markdown

```diff
@@ -946,7 +946,7 @@ mymap.insert(pair<char,int>('Z',26));
 // To iterate
 map<char,int>::iterator it;
 for (it=mymap.begin(); it!=mymap.end(); ++it)
-    std::cout << it->first << "->" << it->second << std::cout;
+    std::cout << it->first << "->" << it->second << '\n';
```

Comment:
I'd prefer we have `std::endl` instead

Reply:
Thanks for the feedback! I just changed it.

Reply:
That was fast :)